### ISSUE TITLE
rive_common's Gradle plugin version needs to be bumped to  1.6.20

### ISSUE THREAD

**Issue by gauravpatil1** [rive-app/rive-flutter#405]
### Description
Getting below error when using below mentioned rive and flutter versions

```
The Android Gradle plugin supports only kotlin-android-extensions Gradle plugin version 1.6.20 and higher.
The following dependencies do not satisfy the required version:
project ':rive_common' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10
```

### Steps To Reproduce
1. Use flutter version 3.22.2 with Dart version  3.4.3
2. Add rive dependency - rive: ^0.13.9
3. rive uses dependency - rive_common: ^0.4.9
4. Use rive in the project
5. Try to build or run the app. You will get below error as rive_common: ^0.4.9 is using Gradle Plugin Version 1.6.10 

```
The Android Gradle plugin supports only kotlin-android-extensions Gradle plugin version 1.6.20 and higher.
The following dependencies do not satisfy the required version:
project ':rive_common' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10
```

### Device & Versions
- Device: Android Emulator
- OS: Android SDK API Level 33
- Flutter Version: 
```
Flutter 3.22.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 761747bfc5 (4 weeks ago) • 2024-06-05 22:15:13 +0200
Engine • revision edd8546116
Tools • Dart 3.4.3 • DevTools 2.34.3
```

**Comment by aniket4390** [comment]
facing same issue

**Comment by HayesGordon** [comment]
This should now be resolved in v0.13.13

Please reopen if you're still experiencing issues after upgrading.
